pkg/sql/catalog/descpb/structured.go

```diff
@@ -145,6 +145,28 @@ func (c ColumnIDs) Equals(input ColumnIDs) bool {
 	return true
 }
 
+// ElementWiseEquals returns true if the input list's elements are the same as this list's elements
+func (c ColumnIDs) ElementWiseEquals(input ColumnIDs) bool {
```

Comment:
nitpick: `WiseEquals` is a bit vague for me.
we have "Equivalent" elsewhere, maybe that's the right name to use.

Reply:
Changed